app/src/lib/stores/git-store.ts

```diff
@@ -1110,7 +1110,8 @@ export class GitStore extends BaseStore {
           currentBranch,
           status.currentUpstreamBranch || null,
           branchTipCommit,
-          BranchType.Local
+          BranchType.Local,
+          ''
```

Comment:
**If** we accept it's valid not having a proper `ref`, I'd say we state that clearly by making `ref` be of type `string | null`.
Therefore this should be…
```suggestion
          null
```

And we should mention that in `ref`s documentation.

Reply:
Updated here to have a valid ref